CHANGELOG.md

```diff
@@ -65,6 +65,7 @@ Querying with using `{__mimir_storage__="ephemeral"}` selector no longer works.
 * [CHANGE] Move auto-scaling panel rows down beneath logical network path in Reads and Writes dashboards. #4049
 * [CHANGE] Make distributor auto-scaling metric panels show desired number of replicas. #4218
 * [CHANGE] Alerts: The alert `MimirMemcachedRequestErrors` has been renamed to `MimirCacheRequestErrors`. #4242
+* [CHANGE] Make use stable `policy/v1` in distributed helm chart. #4337
```

Comment:
This needs to go in the helm Changelog https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/CHANGELOG.md